classes/services/PKPNavigationMenuService.inc.php

```diff
@@ -516,4 +516,41 @@ private function _hasNMTreeNMIAssignmentWithChildOfNMIType($navigationMenu, $nav
 
 		return false;
 	}
+
+	public function _callbackHandleCustomNavigationMenuItems($hookName, $args) {
+		$request = \Application::getRequest();
+
+		$page =& $args[0];
+		$op =& $args[1];
+
+		// Construct a path to look for
+		$path = $page;
+		if ($op !== 'index') $path .= "/$op";
+		if ($ops = $request->getRequestedArgs()) $path .= '/' . implode('/', $ops);
```

Comment:
I'd suggest changing the terminology here -- "operation" / "op" is not the same thing as the requested arguments; use `$args` or `$arguments` for consistency.

Reply:
Fixed